### DISCUSSION TITLE
Cannot find config.yml on digitalocean installation of wiki.js

### DISCUSSION THREAD

**Discussion by astorun** [requarks/wiki#6486] [Help / Questions] [answered]
Hi all,

I purchased and installed wiki.js droplet on my digitalocean account.
However I cannot find config.yml file when I accessed from SSL terminal.
Folder structure is really complicated and I simply fail to locate wiki root folder.

Is there any easier way to find and edit that file like doing it within admin panel.

Thanks.

**Comment by NGPixel** [comment, marked as answer]
Because it's running in a docker container? Not on the host...

```
docker logs wiki
```
